fluent-langneg/test/langneg_filtering_test.js

```diff
@@ -0,0 +1,153 @@
+import assert from 'assert';
+import negotiateLanguages from '../src/index';
+
+describe('Basic Language Negotiation without likelySubtags', () => {
+  const nl = negotiateLanguages;
+
+  it('exact match', () => {
```

Comment:
Should these it() statements read as full sentences starting with "it ..."? mocha docs say yes, not sure if there's a special style among the fluent repos.